mllib/src/main/scala/org/apache/spark/ml/classification/OneVsRest.scala

```diff
@@ -19,23 +19,24 @@ package org.apache.spark.ml.classification
 
 import java.util.UUID
 
-import scala.language.existentials
-
 import org.apache.spark.annotation.Experimental
 import org.apache.spark.ml._
 import org.apache.spark.ml.attribute._
+import org.apache.spark.ml.param.shared.HasRawPredictionCol
 import org.apache.spark.ml.param.{Param, ParamMap}
 import org.apache.spark.ml.util.{Identifiable, MetadataUtils}
 import org.apache.spark.mllib.linalg.Vector
-import org.apache.spark.sql.{DataFrame, Row}
 import org.apache.spark.sql.functions._
 import org.apache.spark.sql.types._
+import org.apache.spark.sql.{DataFrame, Row}
 import org.apache.spark.storage.StorageLevel
 
+import scala.language.existentials
+
 /**
  * Params for [[OneVsRest]].
  */
-private[ml] trait OneVsRestParams extends PredictorParams {
+private[ml] trait OneVsRestParams extends PredictorParams with HasRawPredictionCol {
```

Comment:
It will be easier for you if you have OneVsRestParams extend ClassifierParams.  That will include the rawPredictionCol, plus the updated validateAndTransformSchema method.